tests/tck/slowquery/PermissionViaSameService.feature

```diff
@@ -0,0 +1,99 @@
+    And wait 10 seconds
+    # Make sure the record exists
+    When executing query with user test_permission with password test:
+      """
+      USE nba;
+      SHOW QUERIES
+      | YIELD $-.SessionID AS sid, $-.ExecutionPlanID AS eid, $-.DurationInUSec AS dur
+      WHERE $-.DurationInUSec > 1000000 AND $-.`Query` CONTAINS "GO 100001 STEPS";
+      """
+    Then the result should be, in order:
+      | sid   | eid   | dur   |
+      | /\d+/ | /\d+/ | /\d+/ |
+    When executing query with user test_permission with password test:
+      """
+      USE nba;
+      SHOW QUERIES
+      | YIELD $-.SessionID AS sid, $-.ExecutionPlanID AS eid, $-.DurationInUSec AS dur
+      WHERE $-.DurationInUSec > 1000000 AND $-.`Query` CONTAINS "GO 100001 STEPS"
+      | ORDER BY $-.dur
+      | KILL QUERY(session=$-.sid, plan=$-.eid)
+      """
+    Then an PermissionError should be raised at runtime: Only GOD role could kill others' queries.
+    When executing query with user root with password nebula:
+      """
+      USE nba;
+      SHOW QUERIES
+      | YIELD $-.SessionID AS sid, $-.ExecutionPlanID AS eid, $-.DurationInUSec AS dur
+      WHERE $-.DurationInUSec > 1000000 AND $-.`Query` CONTAINS "GO 100001 STEPS"
+      | ORDER BY $-.dur
+      | KILL QUERY(session=$-.sid, plan=$-.eid)
+      """
+    Then the execution should be successful
+
+  Scenario: [slowquery_test_403] Setup slow query
+    When executing query:
+      """
+      CREATE USER IF NOT EXISTS test_permission WITH PASSWORD 'test';
+      GRANT ROLE USER ON nba TO test_permission;
+      """
+    Then the execution should be successful
+    And wait 5 seconds
+    When executing query with user test_permission with password test:
+      """
+      USE nba;
+      GO 100002 STEPS FROM "Tim Duncan" OVER like YIELD like._dst
+      """
+    Then an ExecutionError should be raised at runtime: Execution had been killed
+
+  Scenario: [slowquery_test_404] Kill successful by user test_permission
+    When executing query:
+      """
+      SHOW QUERIES
+      """
+    Then the execution should be successful
+    # Make sure the record exists
+    And wait 15 seconds
+    When executing query with user test_permission with password test:
+      """
+      USE nba;
+      SHOW QUERIES
+      | YIELD $-.SessionID AS sid, $-.ExecutionPlanID AS eid, $-.DurationInUSec AS dur
+      WHERE $-.DurationInUSec > 1000000 AND $-.`Query` CONTAINS "GO 100002 STEPS";
+      """
+    Then the result should be, in order:
+      | sid   | eid   | dur   |
+      | /\d+/ | /\d+/ | /\d+/ |
+    When executing query with user test_permission with password test:
+      """
+      USE nba;
+      SHOW QUERIES
+      | YIELD $-.SessionID AS sid, $-.ExecutionPlanID AS eid, $-.DurationInUSec AS dur
+      WHERE $-.DurationInUSec > 1000000 AND $-.`Query` CONTAINS "GO 100002 STEPS"
+      | ORDER BY $-.dur
+      | KILL QUERY(session=$-.sid, plan=$-.eid)
+      """
+    Then the execution should be successful
```

Comment:
how about verify if the query existed again?
means

```
# should no result set
SHOW QUERIES| WHERE $-.DurationInUSec > 1000000 AND $-.`Query` CONTAINS "GO 100002 STEPS"
```

Reply:
The execution of query will take a long time.
Checking whether the query is running is on line 85.